### PR TITLE
Import VSTS Release data for test dashboard

### PR DESCRIPTION
**Pull request by philipktlin**
Import release and release environment data from VSTS to iotedge dashboard database.

### CODE REVIEW

File: tools/IoTEdgeDevOps/DevOpsLibTest/IoTEdgeReleaseEnvironmentTest.cs [hunk 1]
```diff
@@ -71,19 +73,20 @@ public void TestEquals()
         [Test]
         public void TestGetHashCode()
         {
-            var releaseEnv = new IoTEdgeReleaseEnvironment(3242, 343406, VstsEnvironmentStatus.Queued);
+            var releaseEnv = new IoTEdgeReleaseEnvironment(3242, 343406, "Dumpy", VstsEnvironmentStatus.Queued);
```
Comment (and-rewsmith) [inline tools/IoTEdgeDevOps/DevOpsLibTest/IoTEdgeReleaseEnvironmentTest.cs]: super nit: typo in test

File: tools/IoTEdgeDevOps/DevOpsLib/IoTEdgeReleaseEnvironment.cs [hunk 2]
```diff
@@ -8,28 +8,41 @@ public class IoTEdgeReleaseEnvironment : IEquatable<IoTEdgeReleaseEnvironment>
     {
         readonly int id;
         readonly int definitionId;
+        readonly string definitionName;
         readonly VstsEnvironmentStatus status;
+        readonly DateTime createdOn;
+        readonly DateTime modifiedOn;
```
Comment (and-rewsmith) [inline tools/IoTEdgeDevOps/DevOpsLib/IoTEdgeReleaseEnvironment.cs]: We don't use these, right?

File: tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs [hunk 3]
```diff
@@ -49,9 +70,11 @@ public async Task RunAsync(TimeSpan waitPeriodAfterEachUpdate, CancellationToken
             }
         }
 
-        private async Task<Dictionary<BuildDefinitionId, DateTime>> ImportVstsBuildsDataAsync(BuildManagement buildManagement, string branch, HashSet<BuildDefinitionId> buildDefinitionIds)
+        async Task<Dictionary<BuildDefinitionId, DateTime>> ImportVstsBuildsDataAsync(BuildManagement buildManagement, string branch, HashSet<BuildDefinitionId> buildDefinitionIds)
         {
-            Dictionary<BuildDefinitionId, DateTime> lastUpdatePerDefinition = this.lastUpdatePerBranchPerDefinition.GetIfExists(branch);
+            Console.WriteLine($"Import VSTS builds from branch [{branch}]");
+            Dictionary<BuildDefinitionId, DateTime> lastUpdatePerDefinition = this.buildLastUpdatePerBranchPerDefinition.GetIfExists(branch);
```
Comment (and-rewsmith) [inline tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs]: nit: it could be more obvious that GetIfExists returns a datetime

Comment (philipktlin) [reply tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs]: no, each branch has multiple build definition; and each definition should have its own last update timestamp.

**Pull request merged**